index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <p id="logo"> My Logo </p>
+        <img id="humburger-btn" src="images/icons8-hamburger-menu-64.png" alt="hamburger btn"/>
+    </header>
```

Comment:
Logo

- [ ] Your hamburger button should be wrapped with a `nav` tag like this 
```
<nav>s
    <img id="humburger-btn" src="images/icons8-hamburger-menu-64.png" alt="hamburger btn"/
</nav>
```
- [ ] The logo and hamburger are not correctly aligned equally along the cross-axis. Please see the image below 👇 
![image](https://user-images.githubusercontent.com/84156714/205163940-31d75a38-a968-401d-9dab-a112fede5848.png)

- Also, in the Figma design, there is more space below the social media icons. Kindly add this space.

Your design | Figma Design
--- | ---
![image](https://user-images.githubusercontent.com/84156714/205163060-1297599e-7071-448e-a0a8-c20fb5bbf689.png) | ![image](https://user-images.githubusercontent.com/84156714/205163782-e075ebea-66b8-4a45-93a6-11804bfb1233.png)